conda_build_config.yaml

```diff
@@ -29,6 +29,7 @@ coverage:
   - 4.*
 cudatoolkit:
   - 10.2.*
+  - 11.0.*
```

Comment:
Maybe we wait on this particular line until the cuda11 stuff is ready.

Reply:
Probably a good idea, although shouldn't affect anyone using the build scripts, since cudatoolkit is being set explicitly.

Reply:
yeah I'll leave that up to you two...the defaults are all set to 10.2 at the moment.  I can remove this if you both would like

Reply:
Let's take it out. We can add it back in when we make CUDA 11 the default.